examples/policies/kubernetes/clusterwide/health.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumClusterwideNetworkPolicy
+metadata:
+  name: "cilium-health-checks"
+spec:
+  endpointSelector:
+    matchLabels:
+      'reserved:health': ''
+  ingress:
+    - fromEntities:
+      - world
```

Comment:
Did you test this policy with `enable-policy=always`? I don't think it's enough to allow health checks. It would likely need an egress rule as well and maybe `remote-node` or `health` instead of `world`.

Reply:
With this change, it converted from (A) to (B). Isn't it enough test?

(A)
```
vagrant@k8s1:~/go/src/github.com/cilium/cilium$ sudo cilium bpf policy get 299
POLICY   DIRECTION   LABELS (source:key[=value])   PORT/PROTO   PROXY PORT   BYTES   PACKETS   
Allow    Ingress     reserved:host                 ANY          NONE         7381    80        
Allow    Ingress     reserved:remote-node          ANY          NONE         5384    62   
```

(B)
```
vagrant@k8s1:~/go/src/github.com/cilium/cilium$ sudo cilium bpf policy get 299
POLICY   DIRECTION   LABELS (source:key[=value])   PORT/PROTO   PROXY PORT   BYTES   PACKETS   
Allow    Ingress     reserved:host                 ANY          NONE         7381    80        
Allow    Ingress     reserved:health               ANY          NONE         0       0         
Allow    Ingress     reserved:remote-node          ANY          NONE         5384    62        
Allow    Egress      reserved:health               ANY          NONE         0       0
```

Reply:
So that shows that the BPF policy map contains what the K8s policy dictates, but I don't think what the K8s policy dictates is what we actually want.

On your cluster with that policy and `enable-policy=always`, is `cilium-health status --probe` reporting a healthy status? Happy to discuss on Slack if you need help to define the minimum required policy.